build_web_compilers/lib/src/sdk_js_compile_builder.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:bazel_worker/bazel_worker.dart';
+import 'package:build/build.dart';
+import 'package:build_modules/build_modules.dart';
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as p;
+
+import 'common.dart';
+import 'dev_compiler_builder.dart' show fixSourceMapSources;
+import 'errors.dart';
+
+const _jsSourceMapExtension = '.js.map';
+
+/// A builder which can compile the SDK to JS from kernel.
+class SdkJsCompileBuilder implements Builder {
+  /// The sdk kernel file for the current platform.
```

Comment:
What does "current platform" mean here? Is this sound vs unsound? Maybe we can mention that in the overall class doc comment?